doc/bc/changes-6.11.md

```diff
@@ -0,0 +1,20 @@
+# Backwards compatibility changes
+
+Changes affecting version compatibility with former or future versions.
+
+## Changes
+
+- EZP-26885: Field Type external storage
+  - Field Type external storage gets directly injected with a proper external storage gateway.
```

Comment:
This is misleading isn't it, people will need to migrate as stated in upgrade doc in order for this change to affect them. As in if they stay on old abstract nothing will change or?